instrumentation/jetty/jetty-testing/src/main/groovy/JettyContinuationHandlerTest.groovy

```diff
@@ -45,7 +45,7 @@ abstract class JettyContinuationHandlerTest extends JettyHandlerTest {
           continuation.resume()
         }
       } else {
-        handleRequest(baseRequest, response)
+        JettyHandlerTest.handleRequest(baseRequest, response)
```

Comment:
Why this change is needed?

Reply:
It's not -- IntelliJ introduced it on its own when I was moving the file around and I must have missed it. Fixed.